src/store/middleware/model-poller.ts

```diff
@@ -152,13 +152,16 @@ export const modelPollerMiddleware: Middleware<
           }),
         );
         const jimmVersion = conn.facades.jimM?.version ?? 0;
+        const rebacEnabled = JSON.parse(
```

Comment:
A small thing but this might be better to be named `rebacFlagEnabled` to distinguish it from whether the controller supports it etc.